  
   <launch>
     <node name="netcomm" pkg="bluedragon_netcomm" type="netcomm_receiver" />
   </launch>
